pep/app/utils/dom.ts

```diff
@@ -79,3 +79,17 @@ export function parseXML(input: string): XMLDocument | Error {
         return new Error('Unable to parse xml');
     }
 }
+
+/**
+ * Load the XSLT doc to do the parsing of the xml
+ *
+ * @return {XSLT}
+ */
+export function loadXSLT(this: any): Promise<Document | null> {
+    return new Promise((resolve) => {
+        let request = new XMLHttpRequest();
+        request.open('GET', `/xmlToHtml.xslt`, false);
+        request.send('');
+        resolve(request.responseXML);
+    });
```

Comment:
I meant to ask this way back when it was first implemented, and more curiosity then anything, but how come we went with a native browser XMLHttpRequest (and a non-async, blocking request at that), over just a normal fetch(), which should be able to handle XML responses too?

Also I think this will be a blocker if we ever want to get document rendering working in fastboot, since there is no XMLHttpRequest in node.

Reply:
When I tried it, It didn't work. However I was able to get it to work this time. 

That being said, we wont be able to render in fastboot anyways without changes because `fetch` does not natively handle XML, but we can convert to text and then use a parser to parse the string.

Reply:
yeah good point, from what I'm seeing, to have full fastboot compatibility it looks like we'd have to treat it as text and use a third party xml parsing lib that is node-friendly.